Gemfile

```diff
@@ -9,8 +9,6 @@ gem "decidim", path: "."
 # Uncomment the following line if you want to use decidim-assemblies plugin
 # gem "decidim-assemblies", path: "."
 
-gem "bootsnap", require: false
-
```

Comment:
Because of the occasional segfaults? :+1: from me, I was planning to do this myself.

Reply:
Yeah, I saw it has problems with latest `bundler`. I'm fine with letting apps add it if they want it :+1: